src/main/java/com/rackspacecloud/metrics/tenantroutingservice/services/RoutingServiceImpl.java

```diff
@@ -48,6 +49,11 @@ public TenantRoutes getIngestionRoutingInformation(String tenantId, String measu
 
         Optional<TenantRoutes> routingInfo = routingInformationRepository.findById(tenantIdAndMeasurement);
 
+        if(!routingInfo.isPresent() && readOnly) {
+            LOGGER.info("Could not look up route for tenantId [{}] and measurement [{}]", tenantId, measurement);
+            throw new RouteNotFoundException(routingInfo.toString());
```

Comment:
Please also add `tenantId` and `measurement` information to the string that is passed on to `RouteNotFoundException`, so that we have those information as well in the log.